python/ray/serve/context.py

```diff
@@ -181,10 +181,23 @@ class _RequestContext:
 
 
 _serve_request_context = contextvars.ContextVar(
-    "Serve internal request context variable", default=_RequestContext()
+    "Serve internal request context variable", default=None
 )
 
 
+@DeveloperAPI
+def get_serve_request_context():
```

Comment:
```suggestion
def _get_serve_request_context():
```

this is an internal API, not public at the moment

Reply:
Fixed, thanks for the review.